[Node.js] Style guide fixes for TDBX vale rules (#842)
mongodb/docs-node: source/quick-start/connect-to-mongodb.txt

         node index.js

>       You will see the details of the retrieved movie document in the 
>       command line output:

Reviewer: ```suggestion
      The output includes details of the retrieved movie document:
```

      proper connection string in the ``index.js`` file, and that you loaded the
      sample data.

> After you complete these steps, you will have a working application that
> uses the driver to connect to your MongoDB deployment, runs a query on
> the sample data, and prints out the result.

Reviewer: ```suggestion
After you complete these steps, you have a working application that
uses the driver to connect to your MongoDB deployment, runs a query on
the sample data, and prints out the result.
```
